includes/server-timing/hooks.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Hook callbacks used for Server Timing.
+ *
+ * @package performance-lab
+ *
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+
+/**
+ * Adds server timing to REST API response.
+ *
+ * @since n.e.x.t
+ *
+ * @param WP_REST_Response|WP_Error $response Result to send to the client. Usually a `WP_REST_Response`.
+ * @return WP_REST_Response|WP_Error Filtered response.
+ */
+function rest_post_dispatch_add_server_timing( $response ) {
```

Comment:
This function is essentially part of the performance lab plugin, so should use the proper prefix for the plugin. The `add_filter` call below will need to be updated as well.
```suggestion
function perflab_rest_post_dispatch_add_server_timing( $response ) {
```